README.md

```diff
@@ -203,6 +210,92 @@ default: N/A
 ```
 > Necessary only for paybutton client to interact with sideshift through the server.
 
+#### proSettings
+```
+type: {
+  enabled: boolean,
+  monthsCost: {
+      [key: string]: number
+  },
+  payoutAddress: string,
+  standardDailyEmailLimit: number | "Inf",
+  proDailyEmailLimit: number | "Inf",
+  standardAddressesPerButtonLimit: number | "Inf",
+  proAddressesPerButtonLimit: number | "Inf"
+}
+default: {
+    "enabled": true,
+    "monthsCost": {
+      "1": 10,
+      "3": 20,
+      "6": 30,
+      "12": 50
+    },
+    "payoutAddress":  "ecash:qrf4zh4vgrdal8d8gu905d90w5u2y60djcd2d5h6un"
+    "standardDailyEmailLimit": 5,
+    "proDailyEmailLimit": 100,
+    "standardAddressesPerButtonLimit": 20,
+    "proAddressesPerButtonLimit": "Inf"
+}
+```
+> General configuration for PayButton Pro. Each parameter is described below.
+
+##### proSettings.enabled
+```
+type: boolean
+default: true
+```
+> If the Pro feature should be enabled or hidden.
+
+##### proSettings.monthsCost
+```
+type: {
+[key: string]: number
+}
+default: {
+"1": 10,
+"3": 20,
+"6": 30,
+"12": 50
+}
+```
+> The pricing model for PayButton Pro subscription — [value] USD for [key] months.
+
+##### proSettings.payoutAddress
+```
+type: string
+default: "ecash:qrf4zh4vgrdal8d8gu905d90w5u2y60djcd2d5h6un"
+```
+> The payout address for PayButton Pro subscriptions.
+
+##### proSettings.standardDailyEmailLimit
+```
+type: number | "Inf"
+default: 5
+```
+> How many emails can a standard user send daily.
+
+##### proSettings.proDailyEmailLimit
+```
+type: number | "Inf"
+default: 20
+```
+> How many emails can a PayButton Pro user send daily.
+
+##### proSettings.standardAddressesPerButtonLimit
+```
+type: number | "Inf"
+default: 5
+```
+> How many addresses can a standard Pro user add for a single button.
+
+##### proSettings.proAddressesPerButtonLimit
+```
+type: number | "Inf"
+default: "Inf"
+```
+> How many addresses can a PayButton Pro user add for a single button.
+
```

Comment:
We don't really get any benefit out of putting this stuff into the readme as:

1. Defaults now need to be updated in two places (here and in the config).
2. Example config has all of it.

Pro pricing is TBD.

Reply:
But this 1. and 2. applies to all config settings, no?

Reply:
Right